app/models/external_registries/stop_heling_result.rb

```diff
@@ -45,33 +55,33 @@ def serial_number
       chassis_number || license_plate_number || "absent"
     end
 
-    def to_external_bike
-      attrs = {}.tap do |h|
-        h[:registry_name] = registry_name
-        h[:registry_id] = registry_id
-        h[:registry_url] = registry_url
-        h[:url] = url
-        h[:type] = "bike"
-        h[:status] = "stolen"
-        h[:mnfg_name] = brand
-        h[:frame_model] = brand_type
-        h[:location_found] = location
-        h[:frame_colors] = colors
-        h[:date_stolen] = date_found
-        h[:serial_number] = serial_number
-        h[:debug] = json.map { |e| e.join(": ") }.join("\n")
-        h[:source_name] = source_name
-        h[:source_unique_id] = source_unique_id
-      end
+    def to_external_registry_bike
+      bike = ::ExternalRegistryBike.find_or_initialize_by(
+        external_id: registry_id,
+        external_registry: external_registry,
+        serial_number: serial_number,
+      )
+
+      bike.url = url
+      bike.cycle_type = "bike"
+      bike.status = "stolen"
+      bike.mnfg_name = brand
+      bike.frame_model = brand_type
+      bike.location_found = location
+      bike.frame_colors = colors
+      bike.date_stolen = date_found
+      bike.info_hash = json
```

Comment:
If we're persisting this in the DB, can we skip including the original JSON?

Reply:
no prob! dropped in ccc4fa28